deps/molecule/molecule.h

```diff
@@ -0,0 +1,498 @@
+/* Copy from molecule v0.2.3 */
```

Comment:
Some code can be deleted.

When I had time, I will split this file to two files: `molecule.h` and : `molecule-debug.h`, then this crate could only include the `molecule.h`.

Reply:
Should be a separate repo in someday?

Reply:
Molecule can add a sub command or option to dump the `.h` file.